fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -162,11 +162,15 @@ public boolean tryReadLock(long timeout, TimeUnit unit) {
     }
 
     public void readUnlock() {
+        long nanoTime = System.nanoTime();
         this.rwLock.readLock().unlock();
+        LOG.debug("table {} get read lock spent {} ms", getId(), (nanoTime - System.nanoTime()) / 1000000L);
```

Comment:
add  if(LOG.isDebugEnabled)  because this line will print many many times. It may have performance problem

Reply:
should not  add time cost caculate here? especial for doris release version?

Reply:
statistics table lock time, we have a lot of table lock time is very serious